frontend/app/utils/route-utils.ts

```diff
@@ -1,7 +1,11 @@
+import { redirect } from 'react-router';
+
+import { getLanguage } from './i18n-utils';
```

Comment:
For consistency with the rest of the application, this should be `'~/utils/i18n-utils'`.